CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **Bug fixes**:
 
 - Trim fields in replays to their defined maximum length. ([#3706](https://github.com/getsentry/relay/pull/3706))
+- Apply rate limit on child spans when the transaction is rate limited. ([#3713](https://github.com/getsentry/relay/pull/3713))
```

Comment:
We also rate limit the extracted span that represents the transaction itself, which is not a child span.

```suggestion
- Apply rate limit on extracted spans when the transaction is rate limited. ([#3713](https://github.com/getsentry/relay/pull/3713))
```